lib/block-supports/layout.php

```diff
@@ -401,15 +411,23 @@ function gutenberg_render_layout_support_flag( $block_content, $block ) {
 	}
 
 	$global_settings        = gutenberg_get_global_settings();
-	$global_layout_settings = _wp_array_get( $global_settings, array( 'layout' ), null );
-	$used_layout            = isset( $block['attrs']['layout'] ) ? $block['attrs']['layout'] : _wp_array_get( $block_type->supports, array( '__experimentalLayout', 'default' ), array() );
+	$global_layout_settings = isset( $global_settings['layout'] ) ? _wp_array_get( $global_settings, array( 'layout' ) ) : null;
+	if ( isset( $block['attrs']['layout'] ) ) {
+		$used_layout = $block['attrs']['layout'];
+	} else {
+		$used_layout = isset( $block_type->supports['__experimentalLayout']['default'] )
+			? _wp_array_get( $block_type->supports, array( '__experimentalLayout', 'default' ), array() )
+			: array();
+	}
```

Comment:
```suggestion
        $used_layout =  array();
	if ( isset( $block['attrs']['layout'] ) ) {
		$used_layout = $block['attrs']['layout'];
	} else if ( isset( $block_type->supports['__experimentalLayout']['default'] ) )  {
		$used_layout = _wp_array_get( $block_type->supports, array( '__experimentalLayout', 'default' ), array() );
	}
```

Improve readability.